jwst/associations/lib/rules_level2b.py

```diff
@@ -932,7 +932,16 @@ def __init__(self, *args, **kwargs):
                 sources=['visitype'],
                 value='.+wfsc.+',
```

Comment:
The value here should not be '.+wfsc.+' but directly 'prime_wfsc_sensing_control'.